src/java.base/share/classes/jdk/internal/module/ModuleInfo.java

```diff
@@ -189,6 +190,7 @@ private Attributes doRead(DataInput input) throws IOException {
 
         int minor_version = in.readUnsignedShort();
         int major_version = in.readUnsignedShort();
+        boolean previewClassfile = minor_version == ClassFile.PREVIEW_MINOR_VERSION;
```

Comment:
If you rename this to isPreview then it would make the uses further down easier to read.